stories/__tests__/SQFormReadOnlyField.stories.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {composeStories} from '@storybook/testing-react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormReadOnlyField.stories';
+
+const {
+  Default: SQFormReadOnlyField,
+  WithInitialValue: SQFormReadOnlyFieldWithInitialValue
+} = composeStories(stories);
+
+describe('SQFormReadOnlyField Tests', () => {
+  describe('Field Only', () => {
+    it('should render a label and read-only text field', () => {
+      render(<SQFormReadOnlyField size="auto" />);
+
+      const label = screen.getByText(/readonly field/i);
+      expect(label).toBeInTheDocument();
+      expect(label).toHaveTextContent('ReadOnly Field');
+
+      const textField = screen.getByRole('textbox', {name: /readonly field/i});
+      expect(textField).toBeInTheDocument();
+    });
+
+    it('should render with "- -" initial value', () => {
```

Comment:
`'should render "- -" when no initial value provided'`

Something to describe _why_ `- -` should render

Reply:
isn't that what "when no initial value provided" answers?

Reply:
> isn't that what "when no initial value provided" answers?

The test currently reads `'should render with "- -" initial value'`

I'm just trying to provide a suggestion that adds immediate clarity to the test. 

Below, we have a test description of `'ReadOnly Field With Initial Value'`

For this test, we could move it so it's more grouped together with the related test.

Then our test could be `'ReadOnly Field Without Initial Value'`

Just trying to call out that users without tribal knowledge of our `- -` pattern may not get much context from the current test description.

Reply:
I see what you're saying. So arguably I could name that one "should render default "- -" when no initial value provided. I guess what drove me to what I initially did is to group them by the stories. We have one story called "Field Only" and one "ReadOnly Field With Initial Value" which might be better called "ReadOnly Field with Custom Initial Value." In any case, I can make some adjustments to clarify your point. Thanks!